platforms/core-runtime/internal-instrumentation-processor/build.gradle.kts

```diff
@@ -15,7 +15,7 @@
  */
 
 plugins {
-    id("gradlebuild.distribution.api-java")
+    id("gradlebuild.distribution.implementation-java")
```

Comment:
💭 This is not API project... So i changed it to implementationa

Reply:
👍